config/routes.rb

```diff
@@ -223,6 +223,7 @@
 
   # Help
   get 'help' => 'help#faq', :as => 'help'
+  get 'help/:topic' => 'help#faq'
```

Comment:
@salbertson Awesome! Thank you for your help. Hound was not complaining about `maxlen` or `quotmarks` recently although we had some PRs with mixed quotes so I think we successfully disabled those checks.